frontend/class-opengraph.php

```diff
@@ -653,7 +653,7 @@ public function description( $echo = true ) {
 			}
 
 			if ( $ogdesc === '' ) {
-				$ogdesc = trim( strip_tags( term_description() ) );
+				$ogdesc = trim( wp_strip_all_tags( term_description() ) );
```

Comment:
As `wp_strip_all_tags` already trims the content, this does not have to be wrapped again.

Reply:
Nice catch. Added new commit fixing this for all instances of `trim( wp_strip_all_tags() )`.